src/MSBuild/Resources/Strings.resx

```diff
@@ -1,6 +1,66 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
Can you please revert all the spurious changes that VS did to this file? :)
I edit the resx files from vscode to make sure VS does not dirty them up.

Reply:
sure